storage_controller/src/scheduler.rs

```diff
@@ -27,7 +27,7 @@ pub enum MaySchedule {
 }
 
 #[derive(Serialize)]
-struct SchedulerNode {
+pub(crate) struct SchedulerNode {
```

Comment:
I'm really annoyed that I had to make this public, but `ShardTag` is used in the `schedule_shard` public function, so the compiler says the trait has to be public too... I don't see a good reason for that to be the case, but :shrug: